cpp/benchmarks/io/parquet/parquet_reader_input.cpp

```diff
@@ -231,6 +231,35 @@ void BM_parquet_read_chunks(nvbench::state& state, nvbench::type_list<nvbench::e
   state.add_buffer_size(source_sink.size(), "encoded_file_size", "encoded_file_size");
 }
 
+template <data_type DataType>
+void BM_parquet_read_wide_tables_mixed(nvbench::state& state,
+                                       nvbench::type_list<nvbench::enum_type<DataType>> type_list)
+{
+  auto const d_type = get_type_or_group(static_cast<int32_t>(DataType));
+
+  auto const n_col       = static_cast<cudf::size_type>(state.get_int64("num_cols"));
+  auto const num_rows    = static_cast<cudf::size_type>(state.get_int64("num_rows"));
+  auto const cardinality = static_cast<cudf::size_type>(state.get_int64("cardinality"));
+  auto const run_length  = static_cast<cudf::size_type>(state.get_int64("run_length"));
+  auto const source_type = io_type::FILEPATH;
```

Comment:
what are we aiming to evaluate with this benchmark? I'm not sure if `io_type::FILEPATH` and `cudf::io::compression_type::NONE` are the right values.

Reply:
Parquet reader's read throughput for increasingly wide-table

Reply:
decided offline on `io_type::DEVICE_BUFFFER` and `compression_type::NONE`
so that the benchmark is focused on decode performance.